core-web/libs/portlets/edit-ema/portlet/src/lib/store/features/editor/toolbar/withEditorToolbar.ts

```diff
@@ -57,7 +58,8 @@ export function withEditorToolbar() {
 
                 const isExperimentRunning = experiment?.status === DotExperimentStatus.RUNNING;
                 const shouldShowUnlock =
-                    pageAPIResponse?.page.locked && pageAPIResponse?.page.canLock;
+                    computePageIsLocked(pageAPIResponse?.page, store.currentUser()) &&
```

Comment:
Create a variable for `computePageIsLocked(pageAPIResponse?.page, store.currentUser())` called `isPageLocked`

```ts
const isPageLocked = computePageIsLocked(pageAPIResponse?.page, store.currentUser())
```